textdb/textdb-perftest/src/main/java/edu/uci/ics/textdb/perftest/excelsink/ExcelSink.java

```diff
@@ -0,0 +1,164 @@
+package edu.uci.ics.textdb.perftest.excelsink;
```

Comment:
Operators do not belong to the perftest package.
Please move it to the appropriate place under `texdb-exp`.